tutorial/write-custom-widget.md

```diff
@@ -288,5 +290,8 @@ func main() {
 	w.ShowAndRun()
 }
 ```
+The widget should look like this:
+	
+![widgetDemoScreenSmall](https://user-images.githubusercontent.com/94919638/145826575-53ae12fc-e6d0-4976-9037-dbdaaea9f4de.png)
```

Comment:
Please don't reference private user images. If you want to add the image, add it in the PR to this repository.